aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/SnapshotCommand.java

```diff
@@ -82,6 +84,10 @@ public class SnapshotCommand implements Callable<Integer> {
                     + " if specified without parameter: ${FALLBACK-VALUE}",
             defaultValue = "true", fallbackValue = "true")
     private boolean consolidation = true;
+    @CommandLine.Option(names = {"--sleep-duration"},
+            description = "Number of seconds used to refresh the ongoing job status. The default value is: ${DEFAULT-VALUE}",
+            defaultValue = "15")
```

Comment:
Sleep duration in publish to imaging step is not the same default value as this. Could this be put in the "global" options to make it the same everywhere ?

Reply:
I think shouldn't be the same for all of them.
There might be some cases where this should differ from one command to another.
I experienced that and decided to let the user customize the value per command.